src_py/apiServer/transmitter.py

```diff
@@ -28,7 +28,7 @@ def clientsTraining(self):
         
     def updateCSV(self):
         print('Training - Update CSV Phase')
-        response = requests.post(self.updateCSVAddress, data='s1,w1,RunOrWalkTrain_splitted')
+        response = requests.post(self.updateCSVAddress, data='s1,w1,w2,RunOrWalkTrain_splitted')
```

Comment:
I still don't understand why data can't be loaded from json 
Adding/Removing workers from json shouldn't involve updating source files.
@dordor7 @kapelnik